docs/DeveloperGuide.adoc

```diff
@@ -297,69 +297,159 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |operator |open the app |I can dispatch personnel and record the incident
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |operator |view available vehicles |I can dispatch vehicles
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |confused operator	|automatically prevent sending of non available vehicles |I won't be allowed to send occupied vehicles
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |operator |to select a vehicle	|it would be dispatched
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |operator |to contact the dispatched vehicle and confirm it has been selected |it would be dispatched
+
+|`* * *` |operator |an ID to be generated for my summaries |my reports can be tagged for easy search
+
+|`* * *` |operator |to have prompts for fields |I know the information required
+
+|`* * *` |careless operator	|edit the report |I won't have to retype everything
+
+|`* * *` |operator on shift	|to save the case for future retrieval |So that others can reference it locally
+
+|`* *` |operator handling many cases |to quickly find relevant parties |I can submit the incident log
+
+|`* *` |regular operator |to view the phone number |I can contact the caller whenever necessary
+
+|`* *` |regular operator |to view the address |I can dispatch personnel based on proximity to address
+
+|`* *` |operator who likes visual cues |to view the vehicles on patrol on a map |I have a visual on who to dispatch
+
+|`* *` |As an operator |to view the available vehicles in descending order of proximity to site |the vehicle can reach the incident site asap
```

Comment:
Would it be better if the table cell did not repeat the same content as the header?
For instance, "As an operator" could simply be "operator" since the header is "As a"